### PR TITLE
add common ENV vars to the Dockerfile #48

### PR DESCRIPTION
**Pull request by EugenMayer**

### CODE REVIEW

File: Dockerfile.erb [hunk 1]
```diff
@@ -16,6 +16,14 @@ FROM alpine:3.6
 
 MAINTAINER GoCD <go-cd-dev@googlegroups.com>
 
+# Important: this values are yet read only, you cannot and should not change them, rather use them in your scripts
+ENV GOCD_DATA_HOME=/godata
+ENV GOCD_DB_HOME=/godata/db
+ENV GOCD_ADDONS_HOME=/godata/addons
+ENV GOCD_PLUGIN_HOME=/godata/plugins
+ENV GOCD_PLUGIN_EXTERNAL_HOME=/godata/plugins/external
+ENV GOCD_ARTIFACTS_HOME=/godata/artifacts
```
Comment (ketan) [inline Dockerfile.erb:25]: Could you add another couple of vars for `/godata/config` `/godata/logs`? Also, if you could rollback the changes to `Dockerfile`, we can merge this PR.

**Comment by EugenMayer** [comment]
@ketan currently now longer working on this, feel free to add the vars or whatever you need and reroll - i am not able to work on this soonish

**Comment by EugenMayer** [comment]
should i close this PR - i think its outdated / overhauled?

**Comment by arvindsv** [comment]
Looks to me like you're right. @ketan ?

**Comment by ketan** [comment]
@varshavaradarajan would know best

**Comment by varshavaradarajan** [comment]
@EugenMayer - 
Closing this PR. 

I assume that the point of these env vars is so that they are available for some scripts. You mentioned something along the same lines in https://github.com/gocd/docker-gocd-server/issues/48. These directories are internal to GoCD and store GoCD related data there. I don't get the point of how *all* these env vars are useful. I can understand if say, artifacts location is needed for some custom cleanup or handling of artifacts in custom scripts. So, if you or @ketan can update https://github.com/gocd/docker-gocd-server/issues/48 with use cases for some or all of the env vars, we can revisit this. Hope that's okay. 

Thank you for your PRs.

**Pull request closed without merging**